include/exadg/structure/user_interface/parameters.h

```diff
@@ -130,6 +130,9 @@ class Parameters
   // density rho_0 in initial configuration (only relevant for unsteady problems)
   double density;
 
+  // linear weak damping coefficient (mass proportional) for unsteady problems
+  double weak_damping_coefficient;
```

Comment:
In case nonlinear problems should not be supported right now, this needs to be documented here.

Further, an assert would need to be added to Parameters::check().